base/pkg/dir.jl

```diff
@@ -54,8 +55,15 @@ function init(meta::AbstractString=DEFAULT_META, branch::AbstractString=META_BRA
                 close(io)
             end
         end
+        #Move TEMP to METADATA
+        mkpath(dir)
+        Base.mv(joinpath(temp_dir,"METADATA"), metadata_dir)
+        Base.mv(joinpath(temp_dir,"REQUIRE"), joinpath(dir,"REQUIRE"))
+        Base.mv(joinpath(temp_dir,"META_BRANCH"), joinpath(dir,"META_BRANCH"))
+        rm(temp_dir)
```

Comment:
It wouldn't be silent, would it? I'd think `make test-pkg` would show the error pretty quickly. If `dir` already happens to exist then I don't think it would work to move `temp_dir` to `dir`. I don't think we have a `force` kwarg for `mv`.